### PR TITLE
Add version flag

### PR DESCRIPTION
**Pull request by MasloMaslane**
In preparation for release, I added --version flag.
For now the version in `pyproject.toml` is 0.0.1, but I think we should release this as 1.0.0

### CODE REVIEW

**Comment by tonowak** [comment]
Maybe there is a way to get the version from project.toml? I bet someday we'll forget to change the version in the source code.

**Comment by tonowak** [comment]
> For now the version in pyproject.toml is 0.0.1, but I think we should release this as 1.0.0

Sure, it doesn't really matter. Can I assume that you're following semantic versioning?

File: src/sinol_make/__init__.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import argparse, pkg_resources
 from sinol_make import util
 
 def configure_parsers():
 	parser = argparse.ArgumentParser(
 		prog='sinol-make',
 		description='Tool for creating and testing sio2 tasks',
 	)
-	parser.add_argument("-v", "--version", action="version", version="%(prog)s 0.0.1")
+	parser.add_argument("-v", "--version", action="version", version="%(prog)s " + pkg_resources.get_distribution('sinol_make').version)
```
Comment (tonowak) [inline src/sinol_make/__init__.py:9]: Is the `s` in `%(prog)s` on purpose or is it a typo?

Comment (MasloMaslane) [reply src/sinol_make/__init__.py:9]: No, it's there to specify that the `prog` should be converted to string

Comment (tonowak) [reply src/sinol_make/__init__.py:9]: Ok, I see. So the output will be `sinol-make: 0.0.1`?

**Comment by MasloMaslane** [comment]
I added automatic detection of the version from package info b6857cac3bf6f7a2bdafda9e46a98fffe3b7fd87

**Review by tonowak — APPROVED** [review]

**Comment by MasloMaslane** [comment]
And I'm planning to follow the semantic versioning

**Pull request merged**